app.js

```diff
@@ -1,25 +1,44 @@
 require('./services/database');
 
 const config = require('config');
+const connectMongo = require('connect-mongo');
+
 const express = require('express');
+const mongoose = require('mongoose');
+const session = require('express-session');
 
+const usersRoute = require('./routes/users-route');
 const assetsRoute = require('./routes/assets-route');
 const googleRoute = require('./routes/google-route');
 const passport = require('./services/passport');
 
 const app = express();
+const MongoStore = connectMongo(session);
 
 /**
  * Middleware
  */
+// Store the session on mongo
```

Comment:
Curious, is mongo a good use for storing more temporary information? I don't know much about how it performs and I'm probably more spoiled by frameworks that are already set up for quick storage, but also arguable speed isn't too much of a factor here.

If we do want the option to retain the option to use something faster for fleeting data like this later, perhaps something that ues mongo for now, but have the option of redirecting somewhere else later if we need?

Reply:
So we could have use redis or memcache (though probably not as good). Mongo performs really well, specially on just key value pairs. Probably not as good as redis cause redis is on pure memory, but still pretty good for a small project like this. 

I also didn't wanna setup a redis instance on top of everything else, but it's something we can evaluate when the time comes.